official/transformer/v2/transformer_main_test.py

```diff
@@ -75,23 +85,26 @@ def test_train_static_batch(self):
     t = tm.TransformerTask(FLAGS)
     t.train()
 
+  @unittest.skipUnless(tf.test.is_built_with_cuda(), 'requires GPU')
```

Comment:
actually i wanted this test to run even if there is no GPU. the helper method will select "one_device" with CPU if no GPUs are available. maybe we can rename the test?